### PR TITLE
Moves header subnav to file

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd & @kluo - moving our help subnav fork to a separate file

### CODE REVIEW

**Comment by kluo** [comment]
+1

**Comment by stvstnfrd** [comment]
@caesar2164 Please make sure you're testing these before firing off PRs...

File: cms/templates/widgets/header_nav_help_post.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="wrapper wrapper-nav-sub">
+    <div class="nav-sub">
+        <ul>
+            <li class="nav-item nav-help-documentation">
+                <a href="https://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/index.html">${_("Studio Documentation")}</a>
```
Comment (stvstnfrd) [inline cms/templates/widgets/header_nav_help_post.html]: You need to import `_`.

**Comment by caesar2164** [comment]
retest this please

**Comment by stvstnfrd** [comment]
retest this please

**Pull request merged**